BlogTemplate/Pages/Drafts.cshtml.cs

```diff
@@ -21,10 +21,11 @@ public DraftsModel(BlogDataStore dataStore)
         }
 
         public IEnumerable<Post> Posts { get; set; }
+        public IEnumerable<Post> PrivatePosts { get; set; }
 
         public void OnGet()
         {
-            Posts = _dataStore.GetAllPosts();
+            PrivatePosts = _dataStore.GetAllPosts().Where(post => !post.IsPublic);

```

Comment:
I prefer the name PrivatePosts because that's really what's in there. It's not "just a list of posts" - it's specifically PrivatePosts (or DraftPosts, or NonPublicPosts).